it/uk/gov/hmrc/agentrelationships/connectors/DesConnectorSpec.scala

```diff
@@ -261,6 +283,12 @@ class DesConnectorSpec
       await(desConnector.deleteAgentRelationship(Vrn("foo"), Arn("bar"))).processingDate should not be null
     }
 
+    "delete relationship between agent and client and return 200 for Utr service" in {
+      givenAgentCanBeDeallocatedInDes(Vrn("foo"), Arn("bar"))
```

Comment:
shouldn't we use Utr here instead of Vrn?

Reply:
Yep